src/lib/sidebar/EntireScheme.svelte

```diff
@@ -24,6 +28,10 @@
 
   let loaded = false;
   let displayClearAllConfirmation = false;
+  let scheme: SchemeData;
```

Comment:
`$: scheme = getFirst($gjSchemeCollection)` is equivalent and handles cleaning up the subscription. But... I need to think about this a bit more; this will run constantly every single time anything in the app does anything. It's only meant to handle loading a new file, clearing everything, etc?

Reply:
What you've said presumably still works but the reason I have to do this probably goes away with the next piece of work: it's so that when you update the scheme in the pipeline form the name update gets propagated

Reply:
(Which presumably is resolved more nicely when we properly support multiple schemes on sketch page)